{"name": "Проверка запроса заказа с неверным трек-номером", "status": "passed", "start": 1712139103394, "stop": 1712139103508, "uuid": "39a74075-0c4e-44d0-9777-91e6d53abdd3", "historyId": "7d8021caaa29b8acfbd67f84085ea950", "testCaseId": "7d8021caaa29b8acfbd67f84085ea950", "fullName": "tests.orders.test_get_order_by_track.TestGetOrderByTrack#test_get_order_wrong_track_number", "labels": [{"name": "parentSuite", "value": "tests.orders"}, {"name": "suite", "value": "test_get_order_by_track"}, {"name": "subSuite", "value": "TestGetOrderByTrack"}, {"name": "host", "value": "MacBook-Air-2.local"}, {"name": "thread", "value": "35467-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.orders.test_get_order_by_track"}]}